New README (#12)
JaneliaSciComp/JaneliaDataStandards: README.md

Reviewer: > "...since Janelia's experiments will always be one step ahead of community conventions"
Might be a little presumptuous.

# Janelia Data Standards

## Purpose

> The Janelia Data Standards group was formed by bioimaging developers who have encountered specific, practical bioimaging data dilemmas that international community standards don’t yet cover. When cutting-edge experiments produce new kinds of data or metadata, developers may be forced to modify existing file formats, or occasionally create new ones. For example, it is often unclear how to store and describe high dimensional images, image tiles that form parts of a whole, or images accompanied by large ancillary data sets. Even though developers may be making reasonable choices on their own, a chaotic landscape can result if they do not communicate their choices, and encourage others to store similar data in a similar way.

Reviewer 2: ```suggestion
The Janelia Data Standards group was formed by bioimaging developers who have encountered specific, practical bioimaging data dilemmas for which there is little or no guidance from international community standards. When cutting-edge experiments produce new kinds of data or metadata, developers may be forced to modify existing file formats, or occasionally create new ones. For example, it is often unclear how to store and describe high dimensional images, image tiles that form parts of a whole, or images accompanied by large ancillary data sets. Even though developers may be making reasonable choices on their own, a chaotic landscape can result if they do not communicate their choices, and encourage others to store similar data in a similar way.
```

Reviewer 2: Sometimes I wonder if we are just repeating what came before in esoteric places such as the Confocal Listserv:
https://lists.umn.edu/cgi-bin/wa?A0=CONFOCALMICROSCOPY

Author: I feel like this is pretty different from that project, which seems like part Q&A forum, part announcements, and isn't curated. Should I add something more to the README to clarify what makes us different? Maybe contrast this project with forums like image.sc?

> **This website is Janelia’s bioimaging developers’ manifesto.** It is a collection of essays written by developers, for developers, on the advanced technical quandaries they’ve encountered. It records the choices Janelia’s developers have made when encountering exotic data dilemmas, so that those encountering similar dilemmas can make consistent choices. Janelia is excited about international standardization efforts, particularly OME-NGFF. Janelia is contributing to OME-NGFF and is rooting for its success. However, such efforts are not enough, since Janelia's experiments will always be one step ahead of community conventions. This manifesto is meant to fill that gap.

Reviewer: > "...since Janelia's experiments will always be one step ahead of community conventions"

Might be a little presumptuous.

Reviewer: How about: 

>"...since Janelia's data needs may often outpace or even diverge from contemporary community standards."

Reviewer 2: "manifesto" has political connotations. Could you explain this choice of word?

Author: @rhoadesScholar I'm fine with your suggestion, or:

> "...since research experiments will always be one step ahead of community conventions."

I wasn't trying to say Janelia is special, just that research, by definition, pushes boundaries.

@mkitti I like 'manifesto' because it implies the work is driven by ideological conviction, and that the principles are meant to be authoritative. I'm open to alternatives, but I don't love the ones that come to mind:
**'blog'** - Feels wishy-washy, like these are just passing observations.
**'guidelines / conventions / standards / style guide'** - These terms feel confusing because they suggest a list of rules, rather than a collection of stories.
But as I say, I'm willing to be persuaded otherwise.

Reviewer 3: ```suggestion
**This website is Janelia’s bioimaging developers’ manifesto.** It is a collection of essays written by developers, for developers, on the advanced technical challenges they’ve encountered. It records the choices Janelia’s developers have made when encountering exotic data, so that those encountering similar situations can make consistent choices. Janelia is excited about international standardization efforts, particularly OME-NGFF. Janelia is contributing to OME-NGFF and is rooting for its success. However, such efforts are not enough, since Janelia's experiments will always be one step ahead of community conventions. This manifesto is meant to fill that gap.
```
The word dilemma (+ synonyms that I was not familiar with) is used a lot. I would probably substitute this with a more straightforward word, so that this manifesto is applicable to a broad set of cases. Being in a dilemma is very subjective.

## Style

The articles in this collection will be vetted, and their conclusions authoritative, for Janelia’s purposes. Where swift, unambiguous decisions are needed, such decisions will be made. The rationale behind those decisions will be explained, and they will become standard practice at Janelia. It is not this group’s mission to write a comprehensive textbook on image data design. Contributors are encouraged to remain pragmatic, to describe their use cases, and to share their example data. That being said, where applicable, developers should speak to the abstract design principles that drove their choices.

## Contributing

> This project aims to develop conventions that Janelians need to do their work, and to disseminate those conventions across Janelia. It is not this group’s goal to create an international standard. However, as this project matures, contributions from the community may be considered. Individuals outside of Janelia who are interested in writing an article should create a GitHub issue to explore this possibility before investing additional time in it. It is this group’s hope that the rapidly evolving conventions developed here may ultimately, gradually, be considered for incorporation into the OME-NGFF standard as well. 

Reviewer 3: ```suggestion
This project aims to develop conventions that Janelians need to do their work, and to disseminate those conventions across Janelia. It is not this group’s goal to create an international standard. However, as this project matures, contributions to and from the community may be considered. Individuals outside of Janelia who are interested in writing an article should create a GitHub issue to explore this possibility before investing additional time in it. It is this group’s hope that the rapidly evolving conventions developed here may ultimately, gradually, be considered for incorporation into the OME-NGFF standard as well. 
```

To highlight that we also want to contribute to international efforts if an idea proves particularly useful and generalizable.

## Product

This effort is in its infancy. Ultimately, the project is expected to consist of four components:

- Written articles. 

> - A cool website that hosts the articles.

Reviewer: --> "A reader accessible website."

> - A cool website that hosts the articles.

Reviewer 2: ```suggestion
- An accessible and easily readable website the hosts the articles.
```
